src/views/ShipmentModal.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-title>{{ $t("Shipment Details") }}</ion-title>
+      <ion-buttons slot="end" @click="closeShipmentModal()" >
+        <ion-button >
+          <ion-icon :icon="closeOutline" />
+        </ion-button>
+      </ion-buttons>  
+    </ion-toolbar>   
+  </ion-header> 
+  <ion-content class="ion-padding">
+    <ion-list>
+      <ion-item>
+        <ion-label position="floating">{{ $t("Street name") }}</ion-label>
+        <ion-input /> 
+      </ion-item> 
+      <ion-item>
+        <ion-label position="floating">{{ $t("House/Apartment number") }}</ion-label>
+        <ion-input /> 
+      </ion-item>     
+      <ion-item>
+        <ion-label position="floating">{{ $t("City") }}</ion-label>
+        <ion-input /> 
+      </ion-item>     
+      <ion-item>
+        <ion-label position="floating">{{ $t("State / Province") }}</ion-label>
+        <ion-input /> 
+      </ion-item>
+      <ion-item>
+        <ion-label position="floating">{{ $t("Zip code") }}</ion-label>
+        <ion-input /> 
+      </ion-item>
+      <ion-item>
+        <ion-label position="floating">{{ $t("Country") }}</ion-label>
+        <ion-select>
+          <ion-select-option>Afghanistan</ion-select-option>
+          <ion-select-option>Albania</ion-select-option>
+          <ion-select-option>India</ion-select-option>
+          <ion-select-option>United States</ion-select-option>   
+        </ion-select>
+      </ion-item>
+    </ion-list>
+    <ion-button @click="closeShipmentModal()">{{ $t("Save") }}</ion-button>
```

Comment:
Improve method name closeShipmentModal based upon it's purpose. Something like save